### PR TITLE
Consume geojson from s3

### PR DESCRIPTION
**Pull request by jeffgrunewald**
This PR will allow Reaper to respond to file:ingest:end events when Odo has completed the conversion of a shapefile into geojson, retrieve the file from S3 and perform a `data:{extract|ingest}:start` event to begin ingesting the geojson into presto.

### CODE REVIEW

File: lib/reaper/data_slurper/s3.ex [hunk 1]
```diff
@@ -0,0 +1,26 @@
+defmodule Reaper.DataSlurper.S3 do
```
Comment (LtChae) [inline lib/reaper/data_slurper/s3.ex]: Does this mean we can set datasets to ingest directly from s3 now as well? i.e. Private datasets that we received as bulk files.

Comment (jeffgrunewald) [reply lib/reaper/data_slurper/s3.ex]: Theoretically, yes, that's correct.

File: lib/reaper/url_builder.ex [hunk 2]
```diff
@@ -9,6 +9,8 @@ defmodule Reaper.UrlBuilder do
   Returns a string containing the URL with all query string parameters based on the `Reaper.ReaperConfig`
   """
   @spec build(SmartCity.Dataset.t()) :: String.t()
+  def build(%SmartCity.Dataset{technical: %{sourceUrl: "s3://" <> _location = url}} = _dataset), do: url
```
Comment (dunson062786) [inline lib/reaper/url_builder.ex]: If you are using location should you remove the underscore?

Comment (jeffgrunewald) [reply lib/reaper/url_builder.ex]: i'm not using the location in this instance; i'm validating that the `url` has the pattern of being a string starting with `s3://` and the some more stuff. as long as it adheres to that pattern, i pass the entire thing through.

**Pull request merged**